CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
+
+Features:
+
+- [Recipes] Add support for classification models to MLflow Recipes (#7082, @bbarnes52)
+- [Tracking] Update `get_model_dependencies` to download all referenced requirements files for specified models (#6733, @harupy)
+- [Tracking] Add support for selecting the Keras model `save_format` used by `mlflow.tensorflow.autolog()` (#7123, @balvisio)
+- [Models] Set `mlflow.evaluate()` status to stable as it is now a production-ready API
+- [Models] Simplify APIs for specifying custom metrics and custom artifacts during model evaluation with `mlflow.evaluate()` (#7142, @harupy)
+- [Models] Correctly infer the positive label for binary classification within `mlflow.evaluate()` (#7149, @dbczumar)
+- [Models] Enable automated signature logging for `tensorflow` and `keras` models when `mlflow.tensorflow.autolog()` is enabled (#6678, @BenWilson2)
+- [Models] Add support for native Keras and Tensorflow Core support within `mlflow.tensorflow` (#6530, @WeichenXu123)
+- [Models] Add support for defining the `model_format` used by `mlflow.xgboost.save/log_model()` (#7068, @AvikantSrivastava)
+- [Scoring] Overhaul the model scoring REST API to introduce format indicators for inputs and support multiple output fields (#6575, @tomasatdatabricks; #7254, @adriangonz)
+- [Scoring] Add support for ragged arrays in model signatures (#7135, @trangevi)
+- [UI] Introduce support for pinning runs within the experiments UI (#7177, @harupy)
+- [UI] Simplify the layout and provide customized displays of metrics, parameters, and tags within the experiments UI (#7177, @harupy)
+- [UI] Simplify run filtering and ordering of runs within the experiments UI (#7177, @harupy)
+- [Java] Add `getModelVersion` API to the java client (#6955, @wgottschalk)
+
+Breaking Changes:
+
+The following list of breaking changes are arranged by their order of significance within each category.
+
+- [Core] Support for Python 3.7 has been dropped. MLflow now requires Python >=3.8
+- [Recipes] MLflow Pipelines is now MLflow Recipes - a framework that enables data scientists to quickly develop high-quality models and deploy them to production
+- [Recipes] `mlflow.pipelines` APIs have been replaced with `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, R, and REST APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
+- [Tracking] Remove deprecated `runs` response field from `Get Experiment` REST API response (#6541, #6524 @dbczumar)
+- [Tracking] Remove deprecated `MlflowClient.download_artifacts` API (#6537, @WeichenXu123)
+- [Tracking] Changed the behavior of environment variable handling for `MLFLOW_EXPERIMENT_NAME` such that the value is always used when creating an experiment. (#6674, @BenWilson2)
+- [Tracking] Modify tracking server configuration to use `--serve-artifacts` by default (#6502, @harupy)
```

Comment:
This is more important than any other Tracking item; can we move it to the top? Also, 

```suggestion
- [Tracking] Update `mlflow server` to run in `--serve-artifacts` mode by default (#6502, @harupy)
```